features/administration.feature

```diff
@@ -23,8 +22,8 @@ Feature: Administrate pupils, classes and teachers
 			| admin    | Georg     | Georgmann  | georgmann@schul-cloud.org | admin@schul-cloud.org | Schulcloud1!  | Schulcloud1!!      |
 
 	@editStudent
-	Scenario Outline: user edits a student
-		Given <userRole> logs in with email '<adminsUsername>' and password '<adminsPassword>'
+	Scenario Outline: As a user, I want to be able to edit a student
+		When <userRole> logs in with email '<adminsUsername>' and password '<adminsPassword>'
 		And '<userRole>' performs first login actions
```

Comment:
Why do we have apostrophes here but not in other lines?

Reply:
Because in this case we pass user Role as a parameter. Its necessary to find defined user in step defs.

Reply:
I agree with Iza because It was agreed that we write the parameters inside ''.